docs/rst/microros_documentation/user_api/user_api_middleware.rst

```diff
@@ -210,3 +210,83 @@ A secondary API is provided to ping the Agent with a specific rmw configuration.
         // micro-ROS Agent is not available
         ...
     }
+
+Error handling
+-------------------------
+
+micro-ROS RMW can be configured to report middleware errors to user space using custom callbacks.
```

Comment:
what happens if this option is disabled and a middleware error occurs?

Reply:
8dc261ea584a6750e9f534c5e6a175f8a26913b4